src/Microsoft.Kusto.ServiceLayer/Connection/ConnectionManager.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections.Concurrent;
+using System.Composition;
+
+namespace Microsoft.Kusto.ServiceLayer.Connection
+{
+    [Export(typeof(IConnectionManager))]
+    public class ConnectionManager : IConnectionManager
```

Comment:
New class to hold all connections separate from the Connection Service. Separating the responsibilities of communicating for connections (`ConnectionService`) and maintaining connections (`ConnectionManager`)